hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
What's the intention of the `gsub!`?  As far as I can see it always returns nil, so nothing in this if block will ever get executed